src/components/map-container.js

```diff
@@ -459,6 +470,22 @@ export default function MapContainerFactory(MapPopover, MapControl) {
           >
             {this._renderDeckOverlay(layersToRender)}
             {this._renderMapboxOverlays(layersToRender)}
+            {/*
+                By placing the editor in this map we have to perform fewer checks for css zIndex
+                and fewer updates when we switch from edit to read mode
+              */}
+            <Draw
+              datasets={datasets}
+              editor={uiState.editor}
+              features={editor.features}
+              isEnabled={isEdit}
+              layers={layers}
+              onDeleteFeature={uiStateActions.deleteFeature}
+              onSelect={uiStateActions.setSelectedFeature}
+              onUpdate={visStateActions.setFeatures}
+              style={{zIndex: isEdit ? 0 : -1}}
+              onToggleFeatureLayer={visStateActions.toggleFeatureLayer}
```

Comment:
I can't find `toggleFeatureLayer` is not defined in vis-state-actions, which causes this error:

<img width="711" alt="Screen Shot 2019-12-05 at 3 39 25 PM" src="https://user-images.githubusercontent.com/3605556/70283281-7339c600-1775-11ea-825f-bd671b252fb5.png">

Reply:
oh i guess it's in geo-operation 2 pr?

Reply:
it's in geo-operation 2 branch

Reply:
I removed it from this PR